### PR TITLE
[15.0][FIX] account_payment_partner: only use bank_account_required in bills

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Forward port of https://github.com/OCA/bank-payment/pull/905.

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

File: account_payment_partner/views/account_move_view.xml [hunk 1]
```diff
@@ -40,8 +40,8 @@
                     '|',('company_id', '=', company_id),('company_id', '=', False)]
                 </attribute>
                 <attribute name="attrs">
-                    {'required': [('bank_account_required', '=', True)], 'readonly':
-                    [('state', '!=', 'draft')]}
+                    {'required': [('bank_account_required', '=', True),('move_type', 'in', ('in_invoice', 'in_refund'))],
+                    'readonly': [('state', '!=', 'draft')]}
```
Comment (pedrobaeza) [inline account_payment_partner/views/account_move_view.xml]: What about invisible?

Comment (MiquelRForgeFlow) [reply account_payment_partner/views/account_move_view.xml]: Yeah, but as it wasn't put, I also don't put it.

Comment (pedrobaeza) [reply account_payment_partner/views/account_move_view.xml]: I think that attribute needs some love too, keeping the standard domain on it

Comment (LoisRForgeFlow) [reply account_payment_partner/views/account_move_view.xml]: I agree with Pedro, you should also put the `'invisible': [('move_type', 'not in', ('in_invoice', 'in_refund', 'in_receipt'))]` which was the standard.

Comment (pedrobaeza) [reply account_payment_partner/views/account_move_view.xml]: I think @alexis-via wanted to use this also for customer invoices. Isn't it?

Comment (MiquelRForgeFlow) [reply account_payment_partner/views/account_move_view.xml]: @LoisRForgeFlow  not true. see https://github.com/OCA/bank-payment/blob/11.0/account_payment_partner/views/account_invoice_view.xml#L34

I put invisible only for entries

Comment (LoisRForgeFlow) [reply account_payment_partner/views/account_move_view.xml]: Fair enough, it does not hurt

**Review by LoisRForgeFlow — APPROVED** [review]

**Pull request merged**